lib/dvb/pmt.cpp

```diff
@@ -572,20 +572,21 @@ int eDVBServicePMTHandler::getProgramInfo(program &program)
 		int autoaudio_aache = -1;
 		int autoaudio_aac = -1;
 		int autoaudio_level = 4;
+		int *autoaudio = NULL;
 
 		std::string configvalue;
 		std::vector<std::string> autoaudio_languages;
 		configvalue = eConfigManager::getConfigValue("config.autolanguage.audio_autoselect1");
-		if (configvalue != "" && configvalue != "None")
```

Comment:
the real value in config.autolanguage.audio_autoselect* is not "None" but "---".